Podcast/Assets.xcassets/home_tab_bar_unselected.imageset/Contents.json

```diff
@@ -1,12 +1,12 @@
 {
-  "info" : {
-    "author" : "zeplin",
-    "version" : 1
-  },
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "home.pdf"
+      "filename" : "iHome.pdf"
```

Comment:
can we rename all these to be more descriptive .. because hopefully its the last time we change them! so .. `tabbar_home_icon` or something similar

Reply:
Oops I didn't realize I had to change the name of the file too instead of just the imageset